src/clj/rems/db/applications.clj

```diff
@@ -46,9 +47,12 @@
                    :rems.workflow.dynamic/approved
                    :rems.workflow.dynamic/rejected
                    :rems.workflow.dynamic/returned}
-                 (:event e)) ;; definitely not by applicant
-      (and (= :event/closed (:event e)) (not= (:applicantuserid app) (:actor e))) ;; not by applicant
-      (and (= "close" (:event e)) (not= (:applicantuserid app) (:userid e))))) ;; not by applicant
+                 (or (:event/type e)
+                     (:event e))) ;; definitely not by applicant
```

Comment:
a) now it looks like "definitely not by applicant" refers to `(:event e)`. perhaps move this comment before the `(contains ...)` invocation
b) is `:event` for old-style events? this might warrant a comment